modules/installation-installing-bare-metal.adoc

```diff
@@ -67,6 +67,24 @@ INFO It is now safe to remove the bootstrap resources
 The command succeeds when the Kubernetes API server signals that it has been
 bootstrapped on the control plane machines.
 
+. Show the state of bare metal host worker nodes:
++
+----
+oc get bmh -A
+----
++
+.Example output
+[source,terminal]
+NAMESPACE               NAME              STATE         CONSUMER                      ONLINE 
```

Comment:
nit: add two spaces before `STATE` to align the columns

Reply:
Done